frontend/src/js/components/auditlogs/AuditlogsView.jsx

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import React from 'react';
+
+import { Button } from '@mui/material';
+
+import { InfoHintContainer } from '@northern.tech/common-ui/info-hint';
+import Loader from '@northern.tech/common-ui/loader';
+
+import historyImage from '../../../assets/img/history.png';
+
+export const AuditlogsView = ({
+  selectionState,
+  hasAuditlogs,
+  csvLoading,
+  createCsvDownload,
+  InfoHintComponent = null,
+  NoAuditlogsComponent = null,
+  AuditLogsFilter,
+  children: auditLogsList
```

Comment:
`children` should not assume what they are, but just be wrapped by their parent (here: `AuditlogsView`) - so keeping them as `children` here
```suggestion
  children
```